hack/tools/Makefile

```diff
@@ -29,6 +51,18 @@ OS := $(shell go env GOOS)
 $(BIN_DIR):
 	mkdir -p $@
 
+$(SHARE_DIR):
+	mkdir -p $@
+
+.PHONY: $(GTAR)
+$(GTAR):
+	@$(GTAR) --version > /dev/null || (echo Install GNU Tar with brew install gnu-tar && exit -1)
+
+
+GO_APIDIFF := $(BIN_DIR)/go-apidiff
```

Comment:
I don't think we need GO_APIDIFF

Reply:
@jichenjc I think this is still open

Reply:
updated with latest update (only remove those lines), thanks
@sbueringer